final_project.py

```diff
@@ -0,0 +1,67 @@
+from flask import Flask, request, jsonify
+
+app = Flask(__name__)
+#change for pr
+JSON_NUM = 0
+tasks = {}
+
+@app.route("/tasks", methods=["GET"])
+def getTasks():
+    return jsonify(tasks), 200
+
+@app.route("/tasks/<task_id>", methods=["GET"])
+def getTaskId(task_id):
+    global tasks
+    if task_id in tasks:
+        return jsonify(tasks[task_id]), 200  
+    return jsonify({"error": "Task not found"}), 404
+
+@app.route("/tasks/<task_id>/completed", methods=["PUT"])
+def completeTask(task_id):
+    global tasks
+    if task_id in tasks:
+        tasks[task_id]["message"] = "Task marked as completed"
```

Comment:
i dont quite understand the stucture of a task in your code. it has a title AND a message field? or the message is only for the REST response?